code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -169,6 +169,10 @@
 			if(internals)
 				internals.icon_state = "internal0"
 			internal = null
+		if(istype(I, /obj/item/clothing/mask/muzzle/tapegag))//snowflakey tape gag. Literally no other place to put this.

```

Comment:
That's the part I hate the most. I just couldn't find any other way to make this work right. Any ideas?